types/src/jwks/rsa.rs

```diff
@@ -33,6 +37,19 @@ impl RSA_JWK {
         }
     }
 
+    pub fn to_poseidon_scalar(&self) -> Result<ark_bn254::Fr> {
+        let mut modulus = base64::decode_config(&self.n, URL_SAFE_NO_PAD)?;
+        modulus.reverse(); // This is done to match the circuit TODO(zkid): finalize the jwk hashing scheme.
```

Comment:
"This is done to match the circuit, which requires the modulus in a verify specific format due to how RSA verification is implemented"